source/features/toggle-files-button.css

```diff
@@ -12,7 +12,8 @@
 	transform: rotate(0.5turn);
```

Comment:
I think we got this wrong. "Down" means "open" while "up" means closed. You can see this in this very page for each collapsable file.

Do you mind dropping `:not()` from this selector in this PR? It's a minor change

Reply:
In PR Files tabs, "down" means *open* while **"right"** means *closed*. So I think this for this feature, it should be "down" for open and "left" for *closed*.

Reply:
You're right. Perhaps we should just use octicon-fold/octicon-unfold instead. In another PR